tests/letstest/scripts/test_leauto_upgrades.sh

```diff
@@ -79,17 +79,6 @@ if [ $(python -V 2>&1 | cut -d" " -f 2 | cut -d. -f1,2 | sed 's/\.//') -eq 26 ];
         echo "Certbot shouldn't have updated to a new version!"
         exit 1
     fi
-    if [ -d "/opt/eff.org" ]; then
-        echo "New directory shouldn't have been created!"
-        exit 1
-    fi
-    # Create a 2nd venv at the new path to ensure we properly handle this case
-    export VENV_PATH="/opt/eff.org/certbot/venv"
-    if ! sudo -E ./letsencrypt-auto -v --debug --version --no-self-upgrade 2>&1 | tail -n1 | grep "^certbot $INITIAL_VERSION$" ; then
```

Comment:
What was this even testing?

Reply:
Good question. I had to go digging to find the answer.

Full answer is at https://github.com/certbot/certbot/pull/5392/files#r160726626 but the short(er) version is when we wrote the code to have certbot-auto rebootstrap on CentOS 6, the script's behavior wasn't great if somehow separate virtual environments existed at both the new and old virtual environment paths. This code was setting up a 2nd virtual environment at the new path to make sure our behavior was sane when both exist and we rebootstrap down below.

How about I add this code back, but set `VENV_PATH` to the old virtual environment path so we can keep this test? I'll make this change and rerun test farm tests to make sure everything still works.

Reply:
I did this. In my opinion, the test seems largely unnecessary, but we currently have it and it may catch unexpected behavior so I feel like we might as well keep it.

I also added back a check for the symlink but only when we're running on CentOS 6 and created the 2nd venv like this.

Reply:
Oh. And I ran test farm tests on these changes and they passed.